pkg/promunifi/usg.go

```diff
@@ -78,9 +78,8 @@ func (u *promUnifi) exportUSG(r report, d *unifi.USG) {
 		{u.Device.TotalTxBytes, prometheus.CounterValue, d.TxBytes, labels},
 		{u.Device.TotalRxBytes, prometheus.CounterValue, d.RxBytes, labels},
 		{u.Device.TotalBytes, prometheus.CounterValue, d.Bytes, labels},
-		{u.Device.NumSta, prometheus.GaugeValue, d.NumSta, labels},
-		{u.Device.UserNumSta, prometheus.GaugeValue, d.UserNumSta, labels},
-		{u.Device.GuestNumSta, prometheus.GaugeValue, d.GuestNumSta, labels},
+		{u.Device.NumSta, prometheus.GaugeValue, d.UserNumSta, append(labels, "user")},
+		{u.Device.NumSta, prometheus.GaugeValue, d.GuestNumSta, append(labels, "guest")},
```

Comment:
`append(slice, item)` always updates/changes `slice`; this can get hairy in the future when someone wants to re-use that variable for another purpose. I'd rather see these things get set at the top, and passed down rather than hiding appends in this block of code.
```
	labels := []string{d.IP, d.Version, d.Model, d.Serial, d.Type, d.Mac, d.SiteName, d.Name}
	labelU := append([]string{"user"}, labels...)
	labelG := append([]string{"guest"}, labels...)
```

Reply:
I'm also willing to be swayed on this. The thing I ran into was slightly different where I would do something like `thing = append(thing[:5], new_thing)` and `thing` would then become equal to `new_thing`. Your approach here is slightly different and probably safe.

Reply:
Yea, that makes sense to me.

Reply:
Well, it makes sense to set them in case they need to be re-used. I prefer post-pending additional labels for consistency. Base set + additional.